lib/src/store.dart

```diff
@@ -135,6 +135,9 @@ class Store extends Stream<Store> with Disposable {
   /// called until that future has resolved.
   ///
   /// If the `Store` has been disposed, this method throws a [StateError].
+  /// Deprecated: 2.9.5
+  /// To be removed: 3.0.0
+  @deprecated
   triggerOnAction(Action action, [void onAction(payload)]) {
```

Comment:
To de-dupe code can this just call triggerOnActionV2?

Reply:
thanks! back to you @jacehensley-wf